include/chemfiles/capi/topology.h

```diff
@@ -230,6 +230,62 @@ CHFL_EXPORT chfl_status chfl_topology_residues_linked(
     bool* result
 );
 
+/// Possible bond orders
+typedef enum {  // NOLINT: this is both a C and C++ file
+    /// Not specified
+    CHFL_BOND_UNKNOWN = 0,
+    /// Single bond
+    CHFL_BOND_SINGLE = 1,
+    /// Double bond
+    CHFL_BOND_DOUBLE = 2,
+    /// Triple bond
+    CHFL_BOND_TRIPLE = 3,
+    /// Quadruple bond (present in some metals)
+    CHFL_BOND_QUADRUPLE = 4,
+    /// Qintuplet bond (present in some metals)
+    CHFL_BOND_QINTUPLET = 5,
+    /// Amide bond (required by some file formats)
+    CHFL_BOND_AMIDE = 254,
+    /// Aromatic bond (required by some file formats)
+    CHFL_BOND_AROMATIC = 255,
+} chfl_bond_order;
+
+/// Add a bond between the atoms at indexes `i` and `j` with bond order `bo` in the `topology`.
+///
+/// @example{tests/capi/doc/chfl_topology/bond_order.c}
+/// @return The operation status code. You can use `chfl_last_error` to learn
+///         about the error if the status code is not `CHFL_SUCCESS`.
+CHFL_EXPORT chfl_status chfl_topology_add_bond_order(
```

Comment:
this function name sounds like it is only adding the bond order, not the bond itself. How about `chfl_topology_add_bond_with_order`? It is a bit long though ...

Reply:
This should be mirrored on chfl_frame too, like chfl_frame_add_bond.

Reply:
Sure thing.